src/test/java/seedu/address/logic/parser/FindProfileCommandParserTest.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_MISSING_KEYWORDS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_OPTION;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.profile.FindProfileCommand;
+import seedu.address.logic.parser.profile.FindProfileCommandParser;
+import seedu.address.model.profile.NameContainsKeywordsPredicate;
+
+public class FindProfileCommandParserTest {
+    /**
+     * All inputs passed to FindProfileCommandParser begins with this string.
+     */
+    private String requiredFlagOption = " " + PREFIX_OPTION + FindProfileCommand.COMMAND_OPTION + " ";
```

Comment:
Might want to standardise if we are placing this within the each command parser tests e.g. `FindProfileCommandParserTest` or within the `CommandParserUtil`. For the Edit and Delete commands, it's placed in the `CommandParserUtil`, but I'm not sure which is a better implementation.

Reply:
Yea I think I will shift it to `CommandParserTestUtil` to keep it more consistent thanks